frontend/src/metabase/containers/SaveQuestionModal.unit.spec.tsx

```diff
@@ -84,15 +90,24 @@ function getQuestion({
 
 const EXPECTED_DIRTY_SUGGESTED_NAME = "Orders, Count, Grouped by Total";
 
-function getDirtyQuestion(originalQuestion) {
-  return originalQuestion
-    .query()
-    .breakout(["field", ORDERS.TOTAL.id, null])
-    .question()
-    .markDirty();
+function getDirtyQuestion(originalQuestion: Question) {
+  return (
+    originalQuestion
+      .query()
+      // @ts-expect-error breakout after query() raises type error
+      .breakout(["field", ORDERS.TOTAL.id, null])
+      .question()
+      .markDirty()
+  );
 }
```

Comment:
I think we prefer casting queries to `StructuredQuery` in cases like that:

```ts
const query = originalQuestion.query() as StructuredQuery;
return query.breakout(...).question().markDirty(); 
```

Reply:
<img width="922" alt="image" src="https://user-images.githubusercontent.com/125459446/235445018-d0d569e3-4226-4864-a29a-e7cf21819709.png">
I don't see `breakout` in the api folder at all. Do you think we miss it?

Reply:
found a solution 🙌